hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/profiling/ConsumerRun.java

```diff
@@ -0,0 +1,5 @@
+package pl.allegro.tech.hermes.consumers.consumer.profiling;
+
+public enum ConsumerRun {
+    EMPTY, PROCESSED, DISCARDED, RETRIED
```

Comment:
not sure about the naming here, `discarded` or `retried` message can also be described as `processed`, in this case we want a message to be `successfuly processed`. Maybe `DELIVERED` is more accurate